src/main/webapp/resources/js/contexts/PagedTableContext.js

```diff
@@ -37,16 +41,21 @@ function AnalysesProvider({ children }) {
   const updateTable = () => {
     setTableState({ ...tableState, loading: true });
 
-    fetchPagedAnalyses({
-      current: tableState.current - 1,
-      pageSize: tableState.pageSize,
-      sortColumn: tableState.column,
-      sortDirection: tableState.order,
-      search: tableState.search,
-      filters: tableState.filters
-    }).then(({ analyses, total }) => {
-      setTableState({ ...tableState, ...{ total, analyses, loading: false } });
-    });
+    axios
```

Comment:
Would it not be better to have this axios call in a js file within the js/apis folder?

Reply:
Good idea, updated in 369ac9a0b60829c75e811f8084b31ef0b929c623